debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/PostgresConnection.java

```diff
@@ -761,6 +761,10 @@ protected boolean isTableType(String tableType) {
     @Override
     protected boolean isTableUniqueIndexIncluded(String indexName, String columnName) {
         if (columnName != null) {
+            if (columnName.length() > 5000) {
+                throw new RuntimeException(
+                    "Unique index exceeded allowed size, indicating a potential DOS attack.");
```

Comment:
Let's remove this snippet - _indicating a potential DOS attack_. Keep it as "Unique index exceeded allowed size"